core/scripts/ocr2vrf/setup_ocr2vrf.go

```diff
@@ -34,6 +34,7 @@ func setupOCR2VRFNodes(e helpers.Environment) {
 	linkEthFeed := fs.String("link-eth-feed", "", "LINK-ETH feed address")
 	confDelays := fs.String("conf-delays", "1,2,3,4,5,6,7,8", "8 confirmation delays")
 	lookbackBlocks := fs.Int64("lookback-blocks", 1000, "lookback blocks")
+	useBatching := fs.Bool("use-batching", false, "use a batching coordinator")
```

Comment:
I think it may be okay to just deploy it - if the user wants to use it they can just paste the command you print out, or they can use the regular consumer if they so desire.

Reply:
SGTM, I want to make sure we don't end up making this script too lengthy but maybe not worth worrying about short-term.